kura/org.eclipse.kura.core/src/main/java/org/eclipse/kura/core/linux/util/LinuxProcessUtil.java

```diff
@@ -268,6 +268,48 @@ public static int getPid(String command, String[] tokens) throws Exception {
             }
         }
     }
+    
+    public static int getPid(int pid) throws Exception {
```

Comment:
since this method is back ported from a previous release, and is public, I would leave it as is to not break something